app/models/support/gds/campaign.rb

```diff
@@ -4,14 +4,22 @@ module Support
   module GDS
     class Campaign
       include ActiveModel::Model
-      attr_accessor :title, :other_dept_or_agency, :signed_campaign, :start_date, :end_date, :description,
-                    :call_to_action, :success_measure, :proposed_url, :site_metadescription, :cost_of_campaign
+      attr_accessor :type_of_site, :has_read_guidance, :has_read_oasis_guidance, :signed_campaign,
+                    :start_date, :end_date, :development_start_date, :performance_review_contact_email,
+                    :government_theme, :description, :call_to_action, :proposed_url, :site_metadescription,
+                    :cost_of_campaign, :ga_contact_email
 
-      validates_presence_of :title, :signed_campaign, :start_date, :end_date, :description,
-                            :call_to_action, :success_measure, :proposed_url, :site_metadescription, :cost_of_campaign
+      validates_presence_of :type_of_site, :has_read_guidance, :has_read_oasis_guidance, :signed_campaign,
+                            :start_date, :end_date, :development_start_date, :performance_review_contact_email,
+                            :government_theme, :description, :call_to_action, :proposed_url, :site_metadescription,
+                            :cost_of_campaign, :ga_contact_email
       validates_date :start_date, on_or_after: :today, on_or_before: :end_date
       validates_date :end_date, on_or_after: :start_date
+      validates_date :development_start_date, on_or_after: :start_date
       validates :proposed_url, format: /((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.campaign.gov.uk?/
```

Comment:
Might need to change this regex because the form now suggests `xxxx.campaign.gov.uk` for campaigns (covered by this regex) and  `xxxx.gov.uk` for bespoke (not covered by this regex).  Perhaps we could use a conditional validator based on the value of `type_of_site`.  But we also state that you can choose something else if you give us reasons so we need some other way of validating it.

Reply:
Changed to make the `campaign.` part optional: 
`/((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.(campaign\.)?gov.uk?/`